scripts/install-deps.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+curl -sSfL "https://raw.githubusercontent.com/Kong/go-atc-router/trunk/build-deps.sh" \
```

Comment:
Merge blocker: use 'main' branch and no forks in the script first.
This script probably belongs in this repo itself perhaps?

Reply:
Will need to update the `go-atc-router` Go module version once it's gone through the code review process too, as we'd want to be pointing to a specific tag too.

Reply:
@javierguerragiraldez Any update?

Reply:
i want to simplify a few things on `go-atc-`router`, especially those that move the library around.  part of the complexity is that the requirements for dynamic vs static linking are different (and both have ugly corners).